terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1226,10 +1239,19 @@ def sq(*words):
             }
             for zone in range(num_zones)
         },
+        'aws_ec2_client_vpn_route': {
+            f'gitlab_{zone}': {
+                'client_vpn_endpoint_id': '${aws_ec2_client_vpn_endpoint.gitlab.id}',
+                'target_vpc_subnet_id': f'${{aws_subnet.gitlab_public_{zone}.id}}',
```

Comment:
We prefer the following notation when interpolating strings in combination of the Terraform syntax,
```suggestion
                'target_vpc_subnet_id': '${aws_subnet.gitlab_public_%s.id}' % zone,
```